src/lando/static_src/legacy/js/components/Timeline.js

```diff
@@ -1,50 +1,38 @@
-    let $timeline = $(this);
+    $('button.cancel-landing-job').on('click', function(e) {
+        var button = $(this);
+        var landing_job_id = this.dataset.landing_job_id;
 
-    // Format timestamps
-    $timeline.find('time[data-timestamp]').formatTime();
-  });
-};
+        button.addClass("is-loading");
+        fetch(`/landing_jobs/${landing_job_id}`, {
+            method: 'PUT',
+            body: JSON.stringify({"status": "CANCELLED"}),
+            headers: {
+                'Accept': 'application/json',
+                'Content-Type': 'application/json',
+                'X-CSRFToken': button.data("csrf-token")
+            },
+        }).then(response => {
+            if (response.status == 200) {
+                window.location.reload();
+            } else if (response.status == 400) {
+                button.prop("disabled", true);
+                button.removeClass("is-danger").removeClass("is-loading").addClass("is-warning");
+                button.html("Could not cancel landing request");
+            } else {
+                button.prop("disabled", true);
+                button.removeClass("is-danger").removeClass("is-loading").addClass("is-warning");
+                button.html("An unknown error occurred");
+            }
+        });
+    });
 
-$('button.cancel-landing-job').on('click', function(e) {
-    var button = $(this);
-    var landing_job_id = this.dataset.landing_job_id;
+    return this.each(function() {
+        let $timeline = $(this);
 
-    button.addClass("is-loading");
-    fetch(`/landing_jobs/${landing_job_id}`, {
-        method: 'PUT',
-        body: JSON.stringify({"status": "CANCELLED"}),
-        headers: {
-            'Accept': 'application/json',
-            'Content-Type': 'application/json'
-        },
-    }).then(response => {
-        if (response.status == 200) {
-            window.location.reload();
-        } else if (response.status == 400) {
-            button.prop("disabled", true);
-            button.removeClass("is-danger").removeClass("is-loading").addClass("is-warning");
-            button.html("Could not cancel landing request");
-        } else {
-            button.prop("disabled", true);
-            button.removeClass("is-danger").removeClass("is-loading").addClass("is-warning");
-            button.html("An unknown error occurred");
-        }
+        // Format timestamps
+        $timeline.find('time[data-timestamp]').formatTime();
     });
-});
-
-$("a.toggle-content,button.toggle-content").on("click", function() {
-    /* A link with the `toggle-snippet` class will hide its parent, and show
-     * any of the parent's siblings. For example:
-     * <div>
-     *  <div>Some content <a href="#" class="toggle-content">toggle</a></div>
-     *  <div>Other content <a href="#" class="toggle-content">toggle</a></div>
-     * </div>
-    */
-    var link = $(this);
-    link.parent().hide();
-    link.parent().siblings().show()
-});
```

Comment:
I think this was deleted, possibly by error? (may have been from the original WIP patch). This handles the merge conflict collapse buttons, and the functionality is still needed. You can trigger this behaviour by creating a situation with a merge conflict.

Reply:
Ah, yep, that deletion was in the patch. I'll restore it and test.